src/ape_geth/providers.py

```diff
@@ -268,9 +270,21 @@ def disconnect(self):
         self._web3 = None  # type: ignore
         self._client_version = None
 
+    def stream_request(self, method, params, iter_path="result.item"):
+        payload = {"jsonrpc": "2.0", "id": 1, "method": method, "params": params}
+        results = ijson.sendable_list()
+        coro = ijson.items_coro(results, iter_path)
+
+        resp = requests.post(self.connection_str, json=payload, stream=True)
```

Comment:
Optional / nit: I would have used `self.uri` instead of `self.connection_str` because the connection_str is the abstract version of the URI and the URI is specific to the provider, as is method.